### ISSUE TITLE
Can't change power level of a 3rd person invited into a DM room

### ISSUE THREAD

**Issue by dbkr** [vector-im/element-web#16453]

**Comment by t3chguy** [comment]
This was done as part of https://github.com/matrix-org/matrix-react-sdk/pull/5413 by direction of design

**Comment by niquewoodhouse** [comment]
Couple of suggestions/comments, just looking randomly at things labelled with needing design.

Might it be more helpful to, by default, give everyone newly invited to a DM those permissions so they can do those things?

I would assume, based on expectations of other products, the vast majority of users would expect to be able to do those things within a DM, whether it's multiple people in there or not.  I think it would feel frustrating to the inviter to, once someone has accepted the invite, go and manage their admin level so they can perform certain actions.  And it would feel frustrating to the invited to have to ask/wait for the inviter to do these things before you can communicate as equals.

Failing that....when you invite someone, it might be more helpful to decide their power level then so at least you can be done in one step, and the invited is less likely to be frustrated.

**Comment by t3chguy** [comment]
> Might it be more helpful to, by default, give everyone newly invited to a DM those permissions so they can do those things?

Keep in mind giving someone the same power level as you is irreversible, so inviting a bot into the room you would never be able to kick it.

**Comment by niquewoodhouse** [comment]
> Keep in mind giving someone the same power level as you is irreversible, so inviting a bot into the room you would never be able to kick it.

I'm guessing we can't currently reliably differentiate bots from people?

**Comment by t3chguy** [comment]
Nope, there's no such distinction, plus in any case you might want to invite in your advisor/assistant but not give them Admin where now they can never be demoted/removed
